data-products-dp0-3/table-access-and-queries.rst

```diff
@@ -110,7 +110,7 @@ The ``DiaSource`` and ``SSSource`` tables are 1:1 and can be joined on the ``dia
 All rows* of the ``SSObject`` table have a match with ``MPCORB`` (but not vice versa),
 and the two tables can be joined on the ``ssObjectId`` column.
 
-*There are exactly 4 rows of the ``SSObject`` table which do not have a match in ``MPCORB``.
+*There are exactly 4 rows of the ``SSObject`` table which do not have a match in ``MPCORB``.*
```

Comment:
This was meant to be a footnote from the paragraph above. I think better to rewrite without the asterisk at all and just say:

```
Almost all rows of the DP0.3 ``SSObject`` table have a match with ``MPCORB`` (but not vice versa),
and the two tables can be joined on the ``ssObjectId`` column.
There are exactly 4 rows of the ``SSObject`` table which do not have a match in ``MPCORB``.

```

Reply:
Oh thanks! This was my mistake, when trying to resolve the github warnings one of them flagged this line as having no closing asterix, I should have read it more carefully to see what that was about. Recommended change has been made.